### DISCUSSION TITLE
Adding datapane integration for Plotly/Bokeh/Altair/DataFrames/Folium etc

### DISCUSSION THREAD

**Discussion by lanthias** [lucharo/nb2medium#16] [Ideas]
Hey! Awesome library!

I saw you might need hosts/renderers for various visualization libraries and pandas DataFrames (I noticed you are currently using gists for dfs). We are an open-source library and free hosting platform called [Datapane](https://docs.datapane.com), which allows users to publish and embed Python visualizations and dataframes online.

We currently support:
- Folium
- Altair
- MPL/Seaborn
- Bokeh
- Plotly
- Pandas DataFrames as interactive tables or styled HTML tables
- JSON
- Code snippets

We have an integration with Medium so these can be embedded into posts, so integration should be pretty straightforward. 

Here are some examples of embedding into Medium:
- https://vickyyy-wu.medium.com/gapminder-cellphone-dataset-visualizations-d18abe800409
- https://andyshirey.medium.com/visualizing-jim-crow-voter-suppression-population-participation-and-the-electoral-college-11d71939ca73
- https://towardsdatascience.com/top-6-python-libraries-for-visualization-which-one-to-use-fe43381cd658
- https://medium.com/analytics-vidhya/how-green-is-canada-in-terms-of-electricity-generation-15ec0da8a556

Happy to help with the integration which should be pretty straightforward using our Python API (users need a free API token to publish on datapane.com).

**Comment by lucharo** [comment]
Hi Leo, and thanks for reaching out. This sounds and looks great and it shouldn't be too hard to code up with an extra `Preprocessor`.

I am not going to be able to work on this for the following two weeks but I shall make the changes then :) Also, PRs are welcome if you're comfortable with `nbdev` and want to get this feature earlier on
